fetch-in-react/my-app/src/UserDetails.tsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable @typescript-eslint/no-unused-vars -- Remove me */
+import { useEffect, useState } from 'react';
+import { UserCard } from './UserCard';
+import type { User } from './Users';
+
+type Props = {
+  userId: number;
+  onCancel: () => void;
+};
+export function UserDetails({ userId, onCancel }: Props) {
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState<unknown>();
+  const [user, setUser] = useState<User>();
+
+  useEffect(() => {
+    getUser(userId);
+  }, [userId]);
+
+  async function getUser(userId: number) {
+    try {
+      const response = await fetch(
+        `https://jsonplaceholder.typicode.com/users/${userId}`
+      );
+
+      if (!response.ok) {
+        throw new Error(`there was an error with retrieving user`);
+      }
+      console.log(response.status);
```

Comment:
Logging this doesn't help very much since all you'll see is a number and not know where it came from. It would be best to add the status in the error message on line 26.